src/steps/interfaces.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { BaseComponentProps } from '../internal/base-component';
+import { type StatusIndicatorProps } from '../status-indicator/internal';
+
+export interface StepsProps extends BaseComponentProps {
+  /**
+   * An array of individual steps
+   *
+   * Each step definition has the following properties:
+   *  * `status` (string) - Status of the step corresponding to a status indicator.
+   *  * `statusIconAriaLabel` - (string) - (Optional) Text alternative for the status icon.
+   *  * `header` (react.ReactNode) - Area next to the status indicator of the step.
```

Comment:
just `(ReactNode)`
```suggestion
   *  * `header` (ReactNode) - Area next to the status indicator of the step.
```

Reply:
I think the description of `header` could be improved too, maybe work with Kashi to align how it will be referred to in the usage guidelines